src/frontend/JasmineGraphFrontEnd.cpp

```diff
@@ -93,6 +92,20 @@ static void start_remote_worker_command(int connFd, bool *loop_exit_p);
 static void sla_command(int connFd, SQLiteDBInterface *sqlite, PerformanceSQLiteDBInterface *perfSqlite,
                         bool *loop_exit_p);
 
+static vector<DataPublisher *> getWorkerClients(SQLiteDBInterface *sqlite) {
+    const vector<Utils::worker> &workerList = Utils::getWorkerList(sqlite);
+    vector<DataPublisher *> workerClients;
+    for (int i = 0; i < workerList.size(); i++) {
+        Utils::worker currentWorker = workerList.at(i);
+        string workerHost = currentWorker.hostname;
+        string workerID = currentWorker.workerID;
```

Comment:
Eventhough we take `workerID` here it is never used.

Reply:
Removed

Reply:
Ok